bin/run-pacta.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+tmp=$(tempfile)
+head -n 211 web_tool_script_2.R >> "$tmp"
```

Comment:
Blocking: Okay, I see that this is to remove running Stress Testing from the process, which is going to make things easier during development. That makes sense, but I don't like hard coding line numbers into scripts like this. 

More generally, don't edit scripts (like R files) as part of the execution process. If you want to  avoid running the ST components, wrap them in an `if` statement in the actual script, and make it conditional on either an R option or an envvar.